12_glob_lab/globlab.txt

```diff
@@ -4,26 +4,29 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 1. Type in a glob pattern to match:
 
   1. All files whose name starts with a lowercase letter
-  Answer: 
+  Answer: [a-z]*.*
```

Comment:
files don't need to have an extension... this will match all files that start with lower case and contain a dot in their name, not all files..